### PR TITLE
Exoscale DBaaS Postgres & MySQL

### PR DESCRIPTION
**Pull request by mweibel**
based on #9. 

@Kidswiss

### CODE REVIEW

File: docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+AppCat allows DevOps engineers to quickly provision a MySQL instance.
+
+== Create a MySQL service
+
+include::user:ROOT:partial$login-in-terminal.adoc[]
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to create an Object Storage bucket
```
Comment (tobru) [inline docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc:11]:
```suggestion
.Example to create a MySQL instance
```

File: docs/modules/ROOT/pages/exoscale-dbaas/postgresql/create.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+AppCat allows DevOps engineers to quickly provision a PostgreSQL instance.
+
+== Create a PostgreSQL service
+
+include::user:ROOT:partial$login-in-terminal.adoc[]
+
+. Apply the following object on your namespace, as specified by its YAML description.
++
+.Example to create an Object Storage bucket
```
Comment (tobru) [inline docs/modules/ROOT/pages/exoscale-dbaas/postgresql/create.adoc:11]:
```suggestion
.Example to create a PostgreSQL instance
```

**Review by tobru — COMMENTED** [review]
It might need some work after #9 has been merged to align the docs.

File: docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc (hunk 1 above)
Comment (mweibel) [reply docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc:11]: huh, how did that come in. sorry - overread. Thanks.

File: docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      majorVersion: "8" <5>
+      mysqlSettings:
+        default_time_zone: "+01:00" <6>
+    size:
+      plan: hobbyist-2 <7>
+  writeConnectionSecretToRef:
+    name: mysql-creds <8>
+----
+<1> Instance name
+<2> The namespace where the object will be created.
```
Comment (tobru) [inline docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc:34]:
```suggestion
<2> The namespace where the object will be created
```

Either all with a `.` at the end or none.

File: docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+<3> Specify backup time of day
+<4> The exoscale zone to use, see xref:references/clouds.adoc#_regions[list of regions]
```
Comment (tobru) [inline docs/modules/ROOT/pages/exoscale-dbaas/mysql/create.adoc:36]:
```suggestion
<4> The Exoscale zone to use, see xref:references/clouds.adoc#_regions[list of regions]
```

File: docs/modules/ROOT/pages/exoscale-dbaas/postgresql/create.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+      majorVersion: "14" <5>
+      pgSettings:
+        timezone: Europe/Zurich <6>
+    size:
+      plan: hobbyist-2 <7>
+  writeConnectionSecretToRef:
+    name: postgres-creds <8>
+----
+<1> Instance name
+<2> The namespace where the object will be created.
```
Comment (tobru) [inline docs/modules/ROOT/pages/exoscale-dbaas/postgresql/create.adoc:34]:
```suggestion
<2> The namespace where the object will be created
```

Dito as for MySQL

File: docs/modules/ROOT/pages/exoscale-dbaas/postgresql/create.adoc [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+<3> Specify backup time of day
+<4> The exoscale zone to use, see xref:references/clouds.adoc#_regions[list of regions]
```
Comment (tobru) [inline docs/modules/ROOT/pages/exoscale-dbaas/postgresql/create.adoc:36]:
```suggestion
<4> The Exoscale zone to use, see xref:references/clouds.adoc#_regions[list of regions]
```

**Review by tobru — APPROVED** [review]

**Pull request merged**